galleries/examples/event_handling/viewlims.py

```diff
@@ -77,17 +87,14 @@ def ax_update(self, ax):
 ax2.imshow(Z, origin='lower',
            extent=(md.x.min(), md.x.max(), md.y.min(), md.y.max()))
 
-rect = UpdatingRect(
-    [0, 0], 0, 0, facecolor='none', edgecolor='black', linewidth=1.0)
-rect.set_bounds(*ax2.viewLim.bounds)
-ax1.add_patch(rect)
-
-# Connect for changing the view limits
-ax2.callbacks.connect('xlim_changed', rect)
-ax2.callbacks.connect('ylim_changed', rect)
-
-ax2.callbacks.connect('xlim_changed', md.ax_update)
-ax2.callbacks.connect('ylim_changed', md.ax_update)
-ax2.set_title("Zoom here")
+rect_selector = RectangleSelector(
+        ax1, select_callback,
+        useblit=True,
+        button=[1],
+        interactive=True,
+        minspanx=5,
+        minspany=5,
+        spancoords='pixels')
 
+ax1.set_title("Zoom here")
```

Comment:
Why did this change Axes to the left one?

That is, if you mean to say, "use the rectangle selector", then maybe rename to "Select here" instead.